docs/en/tutorials/customize_datasetv2.md

```diff
@@ -0,0 +1,464 @@
+# Tutorial 2: Customize Datasets V2
+
+## Data Pre-Processing
+
+### Convert Point cloud format
+
+Currently, we only support bin format point cloud training and inference, before training on your own datasets, you need to transform your point cloud format to bin file. The common point cloud data formats include pcd and las, we provide some open-source tools for reference.
+
+1. Convert pcd to bin: https://github.com/leofansq/Tools_RosBag2KITTI
+2. Convert las to bin: The common conversion path is las -> pcd -> bin, and the conversion from las -> pcd can be achieved through [this tool](https://github.com/Hitachi-Automotive-And-Industry-Lab/semantic-segmentation-editor).
+
+
+### Point cloud annotation
+
+MMDetection3D does not support point cloud annotation. Some open-source annotation tools are offered for reference:
+
+- [SUSTechPOINTS](https://github.com/naurril/SUSTechPOINTS)
+- [LATTE](https://github.com/bernwang/latte)
+
+Besides, we improved [LATTE](https://github.com/bernwang/latte) for better usage. More details can be found [here](https://arxiv.org/abs/2011.10174).
+
+
+## Support new data format
+
+To support a new data format, you can either convert them to existing formats or directly convert them to the middle format. You could also choose to convert them offline (before training by a script) or online (implement a new dataset and do the conversion at training).
+
+
+### Reorganize new data formats to existing format
+
+Once your datasets only contain point cloud file and 3D Bounding box annotations, without calib file. We recommend converting it into the basic formats, the annotations files in basic format has the following necessary keys:
+
+```python
+
+[
+    {'sample_idx':
+     'lidar_points': {'lidar_path': velodyne_path,
+                      ....
+                     },
+     'annos': {'box_type_3d':  (str)  'LiDAR/Camera/Depth'
+               'gt_bboxes_3d':  <np.ndarray> (n, 7)
+               'gt_names':  [list]
+               ....
+            }
+     'calib': { .....}
+     'images': { .....}
+    }
+]
+
+```
+
+In MMDetection3D, for the data that is inconvenient to read directly online, we recommend converting it into basic format and above and do the conversion offline, thus you only need to modify the config's data annotation paths and classes after the conversion.
+For data sharing similar format with existing datasets, like Lyft compared to nuScenes, we recommend directly implementing data converter and dataset class. During the procedure, inheritation could be taken into consideration to reduce the implementation workload.
+
+### Reorganize new data format to middle format
+
+It is also fine if you do not want to convert the annotation format to existing formats.
```

Comment:
There is also a way if users do not want to convert the annotation format to existing formats.